docs/content/any/project/changelogs/NEXT.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2021-05-DD
+menuTitle: 2021-05-DD
+any: true
+description: >-
+  Changelog for Release 2021-05-DD (sqlalchemy-oso 0.6.2) containing new features,
+  bug fixes, and more.
+draft: true
+---
+
+## `sqlalchemy-oso` 0.6.2
+
+### Other bugs & improvements
+
+- Authorized sessions now disable [baked queries][] by default because they can
```

Comment:
I'm not sure this is the right syntax for a link like this.
```suggestion
- Authorized sessions now disable [baked queries]() by default because they can
```

Reply:
For posterity: https://daringfireball.net/projects/markdown/syntax#link

```
The implicit link name shortcut allows you to omit the name of the link,
in which case the link text itself is used as the name. Just use an empty
set of square brackets — e.g., to link the word “Google” to the google.com
web site, you could simply write:

[Google][]

And then define the link:

[Google]: http://google.com/

Because link names may contain spaces, this shortcut even works for multiple
words in the link text:

Visit [Daring Fireball][] for more information.

And then define the link:

[Daring Fireball]: http://daringfireball.net/
```